contentcuration/contentcuration/frontend/channelEdit/components/AddRelatedResourcesModal.vue

```diff
@@ -236,12 +236,12 @@
 
 </script>
 
-<style lang="less" scoped>
+<style lang="scss" scoped>
 
   .list-item-disabled {
     opacity: 0.5;
 
-    /deep/ .v-list__tile--link:hover {
+    ::v-deep .v-list__tile--link:hover {
```

Comment:
For consistency let's wrap these with parenthesis
`::v-deep(.v-list__tile--link:hover)`